phpBB/phpbb/template/twig/tokenparser/event.php

```diff
@@ -18,6 +18,12 @@ class event extends \Twig\TokenParser\AbstractTokenParser
 	/** @var \phpbb\template\twig\environment */
 	protected $environment;
 
+	/** @var \phpbb\event\dispatcher_interface */
+	protected $phpbb_dispatcher;
```

Comment:
Does this need to be a class member? It seems you're only using it once in the constructor and hence I'd remove it